### PR TITLE
Wanhao Duplicator i3 Mini config updates

### PR DESCRIPTION
**Pull request by thisiskeithb**
### Description

Custom bootscreens still aren't shown correctly with `MKS_MINI_12864` defined (See issue https://github.com/MarlinFirmware/Marlin/issues/14889), but this will add a bootscreen in anticipation of that getting fixed. Bootscreen was ported from [Wanhao's Duplicator i3 Mini repo](https://github.com/garychen99/i3Mini).

Also, the LCD contrast is now defined directly in the pins file since the LCD is non-removable (see my comment below).

### Benefits

Adds custom bootscreen & fixes LCD contrast.

### Related Issues

None.

### CODE REVIEW

**Comment by thisiskeithb** [comment]
Per @reallyallnamestaken's [comment](https://github.com/MarlinFirmware/Marlin/issues/12425#issuecomment-520222194), LCD contrast is no longer being set properly. Since the LCD is directly attached to the back of the "[i3 Mini 0ne+ board](https://www.aliexpress.com/item/32849200836.html)" (it's non-removable), the contrast is now set in the pins file.

**Comment by thisiskeithb** [comment]
@thinkyhead Is there a preferred way to handle this controller w/ integrated LCD?

Since it's an all-in-one board with no expandability, I set the contrast directly in the pins file per my comment above, but that causes a ton of warnings when compiling (no errors).

It's also a bit hacky with defining `MKS_MINI_12864` in the config since it doesn't actually have that LCD attached. Maybe `MINIPANEL` would be more appropriate?

https://github.com/MarlinFirmware/Marlin/issues/14174#issuecomment-520269336 provides some insight into defining the LCD contrast to values that work for both `MINIPANEL` and `MKS_MINI_12864`.

**Comment by thinkyhead** [comment]
> Also, the LCD contrast is now defined directly in the pins file 

That's fine if the LCD needs a special value not provided by `Conditionals_LCD.h`.

**Comment by thinkyhead** [comment]
> Is there a preferred way to handle this controller w/ integrated LCD?

Since the LCD is integrated into the board, you can define a unique LCD option that doesn't appear in `Configuration.h`. OTOH, it might just be a more general LCD type with custom settings, in which case that type ought to be enabled along with the board being selected, in the example configs. In that way it would be possible to do a build with the display disabled, if desired.

File: Marlin/src/pins/mega/pins_WANHAO_ONEPLUS.h [hunk 1]
```diff
@@ -92,20 +92,16 @@
 #define KILL_PIN           64
 
 //
-// LCD / Controller
-//
-
-#if HAS_SPI_LCD
```
Comment (thinkyhead) [inline Marlin/src/pins/mega/pins_WANHAO_ONEPLUS.h:98]: These pins are only needed if there is an "SPI LCD" and this is the usual wrapper for these pins. The Wanhao i3 Plus, for example, uses the `MKS_MINI_12864` which produces these defines:

`MKS_MINI_12864` => `MINIPANEL` => `IS_ULTIPANEL` => `ULTIPANEL` `IS_ULTRA_LCD` => `ULTRA_LCD` => `HAS_SPI_LCD`

**Comment by thisiskeithb** [comment]
The latest changes still don't set the LCD contrast correctly/the LCD is unreadable. [Here's a Marlin 1.1.4 config](https://github.com/thisiskeithb/Wanhao-Duplicator-i3-Mini-v1.3-Firmware) from Wanhao that I've unzipped from [Wanhao's original repo](https://github.com/garychen99/i3Mini) & fixed the LCD contrast issue if it'll help troubleshoot what's going on here.

I just tested the `1.1.4` config linked above again and can confirm its setting the LCD contrast to the correct values & the custom Wanhao bootscreen is displayed. It's a fairly hacked up firmware with their custom LCD & custom detection of SD removal/insertion (there's no `SD_DETECT_PIN`) but their implementation works.

Maybe I should have sent you this board as well for debugging 😛

This probably needs a "Needs work"/"Don't merge" tag for the time being while we work through these issue.

**Comment by thinkyhead** [comment]
The main difference I see is that the 1.1.4 configuration is using `MINIPANEL` and not `MKS_MINI_12864`. The `MINIPANEL` supports contrast change, while the `MKS_MINI_12864` does not.

**Comment by thisiskeithb** [comment]
I think I used `MKS_MINI_12864` because it (originally) had a readable contrast without further code mods, but I’ll work on this some more over the weekend. 

Setting up a custom LCD define similar to `MINIPANEL`, but with contrast settings that work for this specific board/LCD will probably be the way to go.

**Comment by thinkyhead** [comment]
> Setting up a custom LCD define similar to MINIPANEL

Is this LCD entirely custom-designed by Wanhao? It seems unusual for a company to design a whole new custom controller instead of using one of the many standard implementations available.

**Comment by thisiskeithb** [comment]
I'm not sure & details are sparse.

I'll do some more testing and report back.

**Comment by thisiskeithb** [comment]
I did some more testing and switching to `MINIPANEL` from `MKS_MINI_12864` resolves the default LCD contrast issue (as expected), so I pushed those changes.

Bootscreen still isn't shown, but that was known from the get-go.

**Pull request merged**